.github/workflows/e2e-test.yml

```
@@ -45,6 +45,7 @@ jobs:
         sudo -s -u $USER --preserve-env bash ${{ github.workspace }}/hack/ci-e2e.sh
 
     - name: Upload artifacts
+      if: always()
```

Per https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions
it should be

```suggestion
      if: ${{ always() }}
```
?

---

Well, I'm pretty sure `always()` will work too, but as the syntax was stated like that, why not follow?

---

I can see few of those too in the ecosystem, but most are using the syntax with curlys, like in the documentation, and in Metal3 we have used the curly syntax, so good for consistency as well.